logic/account.js

```diff
@@ -7,9 +7,12 @@ var jsonSql = require('json-sql')();
 jsonSql.setDialect('postgresql');
 var constants = require('../helpers/constants.js');
 var slots = require('../helpers/slots.js');
+var orderBy = require('../helpers/orderBy.js');
+var BlockReward = require('../logic/blockReward.js');
```

Comment:
inconsistent casing.

should be:
```
blockReward
bignum
```

Reply:
I think we aren't following best practice in the other parts of the code. The best practice is that constructor functions should start with a capital letter. `BlockReward` and `Bignum` are two constructor functions.

Reply:
Ok